store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyStore.java

```diff
@@ -318,7 +318,6 @@ protected Class<ProxyProperties> getPropertiesClass() {
     protected void addAdditionalOperationHandlers() {
         addOperationHandler(OperationChain.class, new OperationChainHandler<>(opChainValidator, opChainOptimisers));
         addOperationHandler(OperationChainDAO.class, new OperationChainHandler<>(opChainValidator, opChainOptimisers));
-        addOperationHandler(GetTraits.class, getGetTraitsHandler());
```

Comment:
removed redundant line.

Reply:
This is not redundant. This handler is not added without this line.